lib/linter/rubocop.rb

```diff
@@ -2,19 +2,24 @@
 
 module Linter
   class Rubocop
+    attr_reader :branch
+    delegate :logger, :to => :branch
+
     def initialize(branch)
       @branch = branch
     end
 
     def run
+      logger.info("#{log_header} Starting Rubocop run...")
       require 'tempfile'
       result = Dir.mktmpdir do |dir|
         @work_dir = File.join(dir, "rubocop")
         Dir.mkdir(@work_dir)
         collect_files
+        logger.info("#{log_header} Collected files #{Dir.glob(File.join(@work_dir, "**/*")).inspect}")
 
         options = {:format => 'json'}
```

Comment:
I think you missed the path to the /tmp directory here...or perhaps missed a `Dir.chdir`